Source/Parallelization/WarpXRegrid.cpp

```diff
@@ -33,11 +33,13 @@ WarpX::LoadBalance ()
 
     // By default, do not do a redistribute; this toggles to true if RemakeLevel
     // is called for any level
-    int doLoadBalance = 0;
+    int loadBalancedAnyLevel = false;
 
     const int nLevels = finestLevel();
     for (int lev = 0; lev <= nLevels; ++lev)
     {
+        int doLoadBalance = false;
```

Comment:
only if Bcast supports this below:
```suggestion
        bool doLoadBalance = false;
```